README.md

```diff
@@ -43,36 +43,39 @@ sudo snap install cos-registration-agent*.snap --dangerous
-                              [--config CONFIG] [--robot-unique-id ROBOT_UNIQUE_ID]
-                              [--log-level {DEBUG,INFO,WARNING,ERROR,CRITICAL}]
+usage: cos-registration-agent [-h] [--config CONFIG] [--url URL] [--shared-data-path SHARED_DATA_PATH] [--grafana-dashboards GRAFANA_DASHBOARDS]
+                              [--foxglove-studio-dashboards FOXGLOVE_STUDIO_DASHBOARDS] [--loki-alert-rule-files LOKI_ALERT_RULE_FILES]
+                              [--prometheus-alert-rule-files PROMETHEUS_ALERT_RULE_FILES] [--log-level {DEBUG,INFO,WARNING,ERROR,CRITICAL}]
                               {setup,update,write-uid,delete} ...
 
 positional arguments:
   {setup,update,write-uid,delete}
                         Action to perform
-    setup               Register device and add custom dashboards device
+    setup               Register device and add custom dashboards
     update              Update custom device data and dashboards
     write-uid           Write device unique ID to a file
     delete              Delete device from server
-options:
+
+optional arguments:
   -h, --help            show this help message and exit
-  --shared-data-path SHARED_DATA_PATH
-                        The path to which the relevant common devices app files
-                        such as robot-unique-id are stored. (default: current_directory)
   --config CONFIG       Config file path. (default: None)
-  --robot-unique-id ROBOT_UNIQUE_ID
-                        Robot unique ID, default set to machine ID. (default: None)
+  --url URL             COS base IP/URL (default: None)
+  --shared-data-path SHARED_DATA_PATH
+                        The path to which the relevant common devices app files such as robot-unique-id are stored. (default:
+                        /home/guillaume/code/rob-cos/cos-registration-agent)
+  --grafana-dashboards GRAFANA_DASHBOARDS
+                        Path to the grafana dashboard (default: None)
+  --foxglove-studio-dashboards FOXGLOVE_STUDIO_DASHBOARDS
+                        Path to the foxglove dashboard (default: None)
+  --loki-alert-rule-files LOKI_ALERT_RULE_FILES
+                        Path to the Loki alert rule files (default: None)
+  --prometheus-alert-rule-files PROMETHEUS_ALERT_RULE_FILES
+                        Path to the Prometheus alert rule files (default: None)
   --log-level {DEBUG,INFO,WARNING,ERROR,CRITICAL}
-                        set the logging level (default: None)
+                        Set the logging level (default: None)
 
-Args that start with '--' can also be set in a config file (specified via
---config). Config file syntax allows:
-key=value, flag=true, stuff=[a,b,c] (for details, see syntax at
-https://goo.gl/R74nmi).
-In general, command-line values override config file values which override
+Args that start with '--' can also be set in a config file (specified via --config). Config file syntax allows: key=value, flag=true,
```

Comment:
so not just tha args that start with '--', but also the actions

Reply:
done